### PR TITLE
[ui][fix] fix mysql timeout

### PR DESCRIPTION
**Pull request by mcada**
//test: @integrations-db-to-db-mysql

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java [hunk 1]
```diff
@@ -121,7 +121,19 @@ public static void cleanUp() {
     public static void waitUntilMysqlIsReady() {
         try {
             OpenShiftWaitUtils.waitUntilPodAppears("mysql");
-            OpenShiftWaitUtils.waitFor(() -> OpenShiftUtils.getPodLogs("mysql").contains("MySQL started successfully"), 300L);
+            OpenShiftWaitUtils.waitFor(() -> OpenShiftUtils.getPodLogs("mysql").contains("MySQL started successfully"), 1000 * 300L);
+            int i = 1;
+            String firstLogs = "";
+            String secondLogs = "a";
+            while (i < 10 && firstLogs.length() < secondLogs.length()) {
```
Comment (tplevko) [inline utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java]: What's the reason behind this? Doesn't the pod log contains some specific notification indicating it's ready?

Comment (mcada) [reply utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java]: Well, yes. That is what first check does. Then some random DB update starts and the test fails as following step is already creating tables. So I came up with this second "smart" check :D

Comment (tplevko) [reply utilities/src/main/java/io/syndesis/qe/templates/MysqlTemplate.java]: OK, seems to be a not the nicest hack, but what can we do...

**Pull request merged**